hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/ReactiveConnection.java

```diff
@@ -60,23 +60,6 @@ interface Expectation {
 
 	CompletionStage<ResultSet> selectJdbc(String sql, Object[] paramValues);
 
-	/**
-	 * This method is intended to be used only for queries returning
-	 * a ResultSet that must be executed outside of any "current"
-	 * transaction (i.e with autocommit=true).
-	 * <p/>
-	 * For example, it would be appropriate to use this method when
-	 * performing queries on information_schema or system tables in
-	 * order to obtain metadata information about catalogs, schemas,
-	 * tables, etc.
-	 *
-	 * @param sql - the query to execute outside of a transaction
-	 * @param paramValues - a non-null array of parameter values
-	 *
-	 * @return the CompletionStage<ResultSet> from executing the query.
-	 */
-	CompletionStage<ResultSet> selectJdbcOutsideTransaction(String sql, Object[] paramValues);
-
```

Comment:
This looks like a breaking change in APIs... ? Should it be documented somewhere?

Reply:
This API is marked as `@Incubating` and I doubt somebody else is implementing, using it. But, I guess I can mark the method as deprecated if we prefer to play it safe.

Reply:
> This API is marked as `@Incubating`

Ah, I didn't see that. Please ignore me then :)